src/metabase/email/messages.clj

```diff
@@ -11,6 +11,10 @@
             [metabase.driver :as driver]
             [metabase.driver.util :as driver.u]
             [metabase.email :as email]
+            [metabase.models :refer [User]]
+            [metabase.models.collection :as collection]
+            [metabase.models.permissions :as perms]
+            [metabase.models.user :as user]
```

Comment:
refer user from `metabase.models.user :as user :refer [User]` rather than import both. Not a huge deal but `User` in `metabase.models` is just a re-varification of the `User` in models.user.